.idea/misc.xml

```diff
@@ -24,7 +24,7 @@
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8 (1)" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```

Comment:
No, please make this work with peroject-jdk-name "1.8 (1)", we don't want to start playing volley ball for every PR raised

Reply:
Actually, I don't remember well but just make it consistent with Ivan's PR, check it there

Reply:
I changed the project-jdk-name to "1.8(1) and its working.

Reply:
With a space before (1) or we will still had a "change"